pkg/printers/internalversion/describe.go

```diff
@@ -1196,6 +1196,14 @@ func describePersistentVolume(pv *api.PersistentVolume, events *api.EventList) (
 		}
 		w.Write(LEVEL_0, "Reclaim Policy:\t%v\n", pv.Spec.PersistentVolumeReclaimPolicy)
 		w.Write(LEVEL_0, "Access Modes:\t%s\n", helper.GetAccessModesAsString(pv.Spec.AccessModes))
+		if len(pv.Spec.MountOptions) == 0 {
+			w.Write(LEVEL_0, "MountOptions:\t<none>\n")
+		} else {
+			w.Write(LEVEL_0, "MountOptions:\n")
+			for _, option := range pv.Spec.MountOptions {
+				w.Write(LEVEL_1, "%s\n", option)
```

Comment:
It should be fine to print all mount options in one line, i.e. `strings.Join(pv.Spec.MountOptions, ", ")`

Reply:
Done, PTAL.